kitchen.yml

```diff
@@ -16,6 +16,7 @@ suites:
     data_bags_path: "test/integration/data_bags"
     run_list:
       - recipe[ros_buildfarm::agent]
+      - recipe[ros_buildfarm::jenkins]
```

Comment:
Instead of running the jenkins and agent recpies together in the agent tests we should create an additional test suite specifically for the jenkins recipe. You can take a peek at #7 which does this for the repo recipe.